packages/grafana-data/src/transformations/standardTransformersRegistry.ts

```diff
@@ -20,11 +20,6 @@ export interface TransformerRegistryItem<TOptions> extends RegistryItem {
    */
   transformation: DataTransformerInfo<TOptions>;
 
-  /**
-   * Optional feature state
-   */
-  state?: PluginState;
```

Comment:
Moved this to the parent `RegistryItem` since it should share all the same semantics